function-images/tests/chained-function-eventing/producer/main.go

```diff
@@ -105,8 +106,7 @@ func main() {
 	defer lis.Close()
 
 	var server server
-	var opts []grpc.ServerOption
-	grpcServer := grpc.NewServer(opts...)
+	grpcServer := grpc.NewServer()
```

Comment:
please add TODO: add tracing support and a link to the Issue (create one if there is none now)